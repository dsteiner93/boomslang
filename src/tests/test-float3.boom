def testfloat(float a, float b) returns void:
	println(a + b)
	println(a - b)
	println(a * b)
	println(a / b)
	println(a == b)
	println(a == a)
	println(a != b)
	println(a != a)
	println(a > b)
	println(a >= b)
	println(a < b)
	println(a <= b)

float c
float d
c = 42.0
d = 3.14159
testfloat(c, d)
testfloat(d, d)
