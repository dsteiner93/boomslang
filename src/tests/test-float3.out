45.14
38.85
131.94
13.36
0
1
1
0
1
1
0
0
6.28
0
9.86
1
1
1
0
0
0
1
0
1
